src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/BuilderUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.builder;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import java.util.Map;
+
+class BuilderUtils {
+
+    static Element buildPropertiesElement(final Map<String,Object> properties, final Document document) {
+        Element propertiesElement = document.createElement("l7:Properties");
+        for (String key: properties.keySet()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Iterate over the "entrySet" instead of the "keySet". ([squid:S2864](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2864))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=52&issues=AWSQdW1usk71k7lwrENu&open=AWSQdW1usk71k7lwrENu)

[ISSUE_KEY]: AWSQdW1usk71k7lwrENu